chapters/sampling-randomization-power.tex

```diff
@@ -1,266 +1,611 @@
+
+Often you will have different levels of units of observation.
+It is common that the unit of observation for analysis
+is different from the unit of observation in the treatment assignment.
+A typical example of this is that a treatment is assigned at the level of a school,
+but data is collected at the level of the students.
+In this case, you need to have one dataset
+measuring student performance
+and another dataset indicating which schools received the treatment.
+You need a way to link those two datasets
+to answer questions about the treatment's impact on student performance.
+
+You might think that you are able to keep all of these details in your head,
+but your whole research team is unlikely to have the same understanding,
+at all times, of all the datasets required.
+The only way to make sure that the full team shares the same understanding
+is to write it down in what is called a \textbf{data map}.\sidenote{
+	\url{https://dimewiki.worldbank.org/data\_map} (TO BE CREATED)}
+\index{data maps}
+
+%% THIS IS A WEIRD 2 PARAGRAPHS WITH JUST A LIST OF QUESTIONS
+A data map is more than just a list of datasets.
+Its purpose is to specify the characteristics and linkages of those datasets.
+Some basic characteristics should filled in during the planning stage.
+Example of those characteristics are:
+What is the unit of observation for each dataset?
+What is the name of the ID variable that uniquely and fully identifies each dataset?
+Should any datasets be possible to merge one-to-one?
+And if so, will the have the same unique identifier?
+Should any datasets be possible to merge many-to-one (for example school and student)?
+And if so, what identifier will be used?
+Solving all of these questions before you acquire any data,
+and making sure that the full research team knows where to find this info,
+will save you a ton of time later on.
+
+Other characteristics should be updated as the project progresses.
+Example of those characteristics are:
+Where will the data be obtain from?
+Where in the project folder are the raw original data stored
+and where are back-ups of the raw data stored?
+See the wiki article linked to above for an template that you can use for your project.
+
+If you have multiple datasets with the same unit of observation
+then you should create a \textbf{master dataset}\sidenote{
+	\url{https://dimewiki.worldbank.org/Master\_Data\_Set}}
+for each of those units of observation and include that in the required datasets.
+The master dataset should include an ID variable
+that can be used to link or merge all datasets with that unit of observation.
+It should also list project wide time-invariant information
+like sampling and treatment status.
+Additionally, any unit of observation
+that will be used in sampling or treatment assignment
+should have a master dataset,
+and that dataset should be used when sampling or assigning treatment.
+
+\section{Taking research design into account when planing your data work}
```

Comment:
think about how to revise this (and previous section) to emphasize project workflow. this can be described in the intro. something like: create data map, assemble master dataset, create treatment identifier (either by randomization or using score or whatnot), select IE sample. in the intro to this section, refer back to this as a stage in the workflow.

Reply:
I like this structure of the chapter.